lib/drag_screen.dart

```diff
@@ -0,0 +1,305 @@
+import 'package:flutter/material.dart';
+import 'dart:io';
+import "dragable.dart";
+import 'dart:math';
+import 'package:image_pixels/image_pixels.dart';
+import "coordinate.dart";
+/*
+How this scruffed ass proof of concept even works
+start from the bottom build function at step 1
+-----------------------------------------------------
+Still needed:
+Still have to draw lines between squares on the screen
+//TODO: Make the blue draggable also update coordinate info
+  //TODO: Make the draggables actually update coordinate information in image handler rather than fake data
+  //TODO: add offset so that point is in middle of draggable
+   //TODO: Make a class that contains the callback functions that can be used for both buttons rather than having a bunch of diff callbacks
+*/
+
+class DragScreen extends StatefulWidget {
+  final String passedImagePath;
+
+  const DragScreen({super.key, required this.passedImagePath});
+
+  //const DragScreen({Key? key}) : super(key: key);
+  static const String id = "drag_screen";
+
+  @override
+  State<DragScreen> createState() => _DragScreenState();
+}
+
+class _DragScreenState extends State<DragScreen> {
+  @override
+  Widget build(BuildContext context) => _DragScreenView(state: this);
+
+  //REFERENCE TO DRAGGABLE WIDGET
+  late Drag_Button draggableOne;
+  late Drag_Button draggableTwo;
+
+  //Passed image from last screen
+  late FileImage imageFile = FileImage(File((widget.passedImagePath)));
+
+  //These X and Y coordinates are the middle of
+  late Coordinate pointInImageResolution;
```

Comment:
You're using one variable to track the location of 2 different draggables. This value gets fixed as the position of the red dot, and the position of the blue dot is never tracked.